ros/python/py_pubsub/py_pubsub/object_detector.py

```diff
@@ -56,6 +57,8 @@ def __init__(self):
         self._detector = ResNetFRCNN(img_batch_size=1, use_cuda=self._use_cuda)
         log.info(f"Ready to detect using {self._detector}")
 
+        self.stop = False
```

Comment:
What is this variable for? Nothing seems to ever set it to anything than this default value. I presumed that the detector node "stop" signal will come in the form of terminating/killing the containing process. Do you have a different use-case in mind?

Reply:
Oops. I was using this to debug. I took it out.